lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -119,32 +119,52 @@ module.exports = [
           details: {
             items: [
               {
-                source: /seo-tester\.html.+:24:12$/,
+                source: {
+                  url: /seo-tester\.html.+$/,
+                  canLink: true,
+                  line: 23,
```

Comment:
the line numbers went down one because the audit no longer adds one to `line`. did this b/c

1) it was a view concern
2) DT expects zero indexed lines for linking.